### PR TITLE
[RW-5529][risk=no] cope disclaimer

### PR DESCRIPTION
**Pull request by mhigbyflowers**
**implementation**
image

**Mockup**
image

### CODE REVIEW

File: public-ui/src/app/data-browser/views/survey-view/survey-view.component.html [hunk 1]
```diff
@@ -1,346 +1,398 @@
...
+          which participants took the survey.</p>
       </div>
     </div>
-    <div class="search-bar-container">
-      <div id="db-search-bar">
-        <clr-icon shape="search" class="is-info search-icon"></clr-icon>
-        <input type="text" aria-label="Survey Search" id="search-db" placeholder="Keyword Search" [formControl]="searchText" style="height:auto" autofocus>
-        <div *ngIf="searchText.value"  class="clear-icon" (click)="clearSearch()"><i class="far fa-times fa-1x clear-search-icon"></i></div>
-      </div>
+    <button *ngIf="copeFlag && domainId == 'covid-19-participant-experience'" (click)="showStatement = true" class="disclaimer-btn">IMPORTANT CONSIDERATIONS FOR COPE SURVEY - LEARN MORE</button>
```
Comment (SrushtiGangireddy) [inline public-ui/src/app/data-browser/views/survey-view/survey-view.component.html]: Filtering on survey concept id is better may be. Incase domain name changes later.

**Comment by SrushtiGangireddy** [comment]
image
Should the button be in purple even when it is not active ? I was just looking at the user journey https://docs.google.com/presentation/d/1dJT8AmpouIpJaaZGmBMVDccQQ68bv8681SWEt-i8N34/edit#slide=id.g960643df0a_1_0.

**Review by SrushtiGangireddy — APPROVED** [review]
Rest looks good to me. Thanks.

**Comment by mhigbyflowers** [comment]
image

**Pull request merged**